supplyChainSimulation/MaterialPlanning_P2.Designer.cs

```diff
@@ -28,6 +28,7 @@ protected override void Dispose(bool disposing)
         /// </summary>
         private void InitializeComponent()
         {
+            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(MaterialPlanning_P2));
```

Comment:
Breaking bug in english version

Reply:
Ist generierter Code für den View, bei mir funktioniert alles. Kannst du das nochmal checken?